StellarApp/Controllers/NewsViewController.swift

```diff
@@ -86,7 +94,15 @@ extension NewsViewController: UICollectionViewDataSource {
         let thisArticle = articles[indexPath.row]
         cell.articleLabel.text = thisArticle.source.name
         cell.articleDescription.text = thisArticle.title
-        ImageHelper.fetchImageFromNetwork(urlString: thisArticle.urlToImage) { (error, data) in
+        
+        if (indexPath.row == articles.count - 1) {
```

Comment:
can you move this to `collectionView willShowCell`?

Reply:
do you mean collectionview willdisplaycell?